Dockerfile

```diff
@@ -16,7 +18,7 @@ RUN pnpm build
 FROM node:18-alpine AS runner
 ENV NODE_ENV production
 # Install necessary tools
-RUN apk add bash postgresql cairo pango jpeg musl giflib pixman pangomm libjpeg-turbo freetype
+RUN apk add bash cairo pango jpeg musl giflib pixman pangomm libjpeg-turbo freetype
```

Comment:
is this change for Vercel? can we make it also compatible with Fly.io? (eg. just ignoring the Dockerfile for Vercel)